aiobs/evals/safety/toxicity_detection.py

```diff
@@ -0,0 +1,427 @@
+        """
+        Create a ToxicityDetectionEval using an OpenAI client.
+
+        Args:
+            client: The OpenAI client instance.
+            model: Model name to use for judging toxicity (default: gpt-4o-mini).
+            **kwargs: Additional configuration options for the evaluator.
+
+        Returns:
+            A configured ToxicityDetectionEval instance.
+        """
+        config = ToxicityDetectionConfig(**kwargs) if kwargs else None
+        return cls(client=client, model=model, config=config)
+
+    @classmethod
+    def with_gemini(
+        cls,
+        client: Any,
+        model: str = "gemini-2.0-flash",
+        **kwargs: Any,
+    ) -> "ToxicityDetectionEval":
+        """
+        Create a ToxicityDetectionEval using a Google Gemini client.
+
+        Args:
+            client: Google GenAI client instance.
+            model: Model name to use for judging toxicity (default: gemini-2.0-flash).
+            **kwargs: Additional configuration options for the evaluator.
+
+        Returns:
+            A configured ToxicityDetectionEval instance.
+        """
+        config = ToxicityDetectionConfig(**kwargs) if kwargs else None
+        return cls(client=client, model=model, config=config)
+
+    @classmethod
+    def with_anthropic(
+        cls,
+        client: Any,
+        model: str = "claude-3-sonnet-20240229",
+        **kwargs: Any,
+    ) -> "ToxicityDetectionEval":
+        """
+        Create a ToxicityDetectionEval using an Anthropic client.
+
+        Args:
+            client: Anthropic client instance.
+            model: Model name to use for judging toxicity 
+                (default: claude-3-sonnet-20240229).
+            **kwargs: Additional configuration options for the evaluator.
+
+        Returns:
+            A configured ToxicityDetectionEval instance.
+        """
+
+        config = ToxicityDetectionConfig(**kwargs) if kwargs else None
+        return cls(client=client, model=model, config=config)
+
+    
+    def _build_prompt(self, eval_input: EvalInput, text: str = None) -> str:
+        """
+        Build the toxicity evaluation prompt.
+
+        Args:
+            eval_input: The input to evaluate.
+            text: Optional override for the text to evaluate (for checking input separately).
+    
+        Returns:
+            Fully formatted toxicity judge prompt string.
+        """
+
+        # Use provided text or default to model output
+        if text is None:
+            text = eval_input.model_output
+
+        # Build the context section only if enabled + context exists
+        context_section = ""
+        if eval_input.context:
+            formatted_ctx = self._format_context(eval_input.context)
+            if formatted_ctx:
+                context_section = f"## Additional Context:\n{formatted_ctx}\n"
+
+        # Construct final prompt
+        prompt = TOXICITY_JUDGE_PROMPT.format(
+            text=text,
+            context=context_section
```

Comment:
```suggestion
            text=text
```

We don't have `context` in `TOXICITY_JUDGE_PROMPT`, so we can remove this.